target/linux/ipq40xx/files/drivers/net/ethernet/qualcomm/essedma/edma.c

```diff
@@ -2023,6 +2023,7 @@ int edma_open(struct net_device *netdev)
 	 */
 	if (adapter->poll_required) {
 		if (!IS_ERR(adapter->phydev)) {
+			genphy_soft_reset(adapter->phydev);
```

Comment:
maybe add the commit description as comment here.

Reply:
Good idea, will do.

Reply:
Done